### PR TITLE
Introduce port group support to ovn-scale-test

### PR DESCRIPTION
**Pull request by LorenzoBianconi**
Introduce port-group support to ovn-scale-test and rely on port-groups configuring ACLs in OpenShift workload

### CODE REVIEW

File: rally_ovs/plugins/ovs/ovsclients_impl.py [hunk 1]
```diff
@@ -216,27 +216,42 @@ def lport_set_options(self, name, *options):
...
-            self.run("acl-del", args=params)
+            self.run("acl-del", opts, args=params)
+
+        def port_group_add(self, port_group, port_list):
+            params = [port_group, port_list]
+            self.run("pg-add", [], params)
+
+        def port_group_set(self, port_group, port_list):
+            params = [port_group, port_list]
+            self.run("pg−set−ports", [], params)
```
Comment (dceara) [inline rally_ovs/plugins/ovs/ovsclients_impl.py]: Hard to spot but this is actually the `−` character while it should actually be `-`.

File: rally_ovs/plugins/ovs/ovsclients_impl.py [hunk 2]
```diff
@@ -216,27 +216,42 @@ def lport_set_options(self, name, *options):
...
+
+        def port_group_del(self, port_group):
+            params = [port_group]
+            self.run("pg−del", [], params)
```
Comment (dceara) [inline rally_ovs/plugins/ovs/ovsclients_impl.py]: Same here: this is actually the `−` character while it should actually be `-`.

File: rally_ovs/plugins/ovs/ovsclients_impl.py (hunk 1 above)
Comment (LorenzoBianconi) [reply rally_ovs/plugins/ovs/ovsclients_impl.py]: @dceara : PR updated, thx

File: rally_ovs/plugins/ovs/ovsclients_impl.py (hunk 2 above)
Comment (LorenzoBianconi) [reply rally_ovs/plugins/ovs/ovsclients_impl.py]: @dceara : PR updated, thx

File: rally_ovs/plugins/ovs/scenarios/ovn_nb.py [hunk 3]
```diff
@@ -62,26 +62,37 @@ def create_or_update_address_set(self, name, ipaddr, create = True):
...
     @atomic.action_timer("ovn.create_port_acls")
-    def create_port_acls(self, lswitch, lports, addr_set):
+    def create_port_acls(self, acl_dev, lports, addr_set,
+                         acl_type="switch"):
         """
         create two acl for each logical port
         prio 1000: allow inter project traffic
         prio 900: deny all
         """
         match = "%(direction)s == \"%(lport)s\" && ip4.dst == %(address_set)s"
```
Comment (numansiddique) [inline rally_ovs/plugins/ovs/scenarios/ovn_nb.py]: It should be this way 

match = "%(direction)s == %(lport)s && ip4.dst == $%(address_set)s"

Please see this PR which I submitted to fix this issue in dceara/ovn-scale-testing repo

https://github.com/dceara/ovn-scale-test/pull/2

Comment (LorenzoBianconi) [reply rally_ovs/plugins/ovs/scenarios/ovn_nb.py]: @numansiddique Thx, updated the PR

**Review by hzhou8 — APPROVED** [review]

**Pull request merged**